src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
add globally for every  element using *

Reply:
after using those settings for every element page looks like that 
![Zrzut ekranu 2024-02-8 o 07 59 06](https://github.com/mate-academy/layout_product-cards/assets/151854090/186a75cb-81ad-4bec-a026-f6f30fa57d2c)

Reply:
i meant globally `box-sizing: border-box`, not flex settings